src/components/switch/index.js

```diff
@@ -124,9 +126,11 @@ class Switch extends BaseComponent {
   renderThumb() {
     const {thumbStyle} = this.getThemeProps();
     const {thumbPosition} = this.state;
+    const output = this.calcThumbOnPosition() * (Constants.isRTL ? -1 : 1);
```

Comment:
I think it would be nicer to move the RTL logic to the `calcThumbOnPosition`, makes it cleaner to read.

Reply:
Done